TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml

```diff
@@ -13,7 +13,7 @@
     <form action="" method="POST" style="margin-top: 50px;">
         <div class="form-group">
             <label style="font-size:20px;" class="control-label" for="CountrySearch"><strong>Enter Country</strong></label>
-            <input type="text" class="form-control" name="CountrySearch" id="CountrySearch" style="width:50%" />
+            <input type="text" class="form-control" placeholder="Please Enter Country to start search..." name="CountrySearch" id="CountrySearch" style="width:50%" />
         </div>
```

Comment:
Placeholder attribute added in search input element for better usage of the search bar space when not populated.

Reply:
Placeholder is a nice trick.  With that, you can remove label.